crates/nu-std/std/mod.nu

```diff
@@ -335,3 +335,16 @@ export def repeat [
 
     1..$n | each { $item }
 }
+
+# return a null device file.
+#
+# # Examples
+#     run a command and ignore it's stderr output
+#     > cat xxx.txt e> (null-device)
+export def null-device []: any -> string {
+    if ((sys).host.name | str downcase) == "windows" {
```

Comment:
i find the `$nu.os-info.name` version easier to read but ehe :wink:

Reply:
Thanks!  And the output of `$nu.os-info.name` seems more stable

Reply:
yup :yum:

Reply:
not even sure we need to `str downcase` it but that's fine :wink: